From 12d819a6bccddfd096b02a407080833d6da902f3 Mon Sep 17 00:00:00 2001
From: Bharat B Panda <bharat.b.panda@intel.com>
Date: Thu, 14 Jul 2022 22:33:54 +0530
Subject: [PATCH] Enable Hostapd aidl HAL

As per Google, from T onwards Hostapd HAL can optionally
use AIDL hall instead of HIDL. It has the required changes
impletemented.

Tracked-On: OAM-102195
Signed-off-by: Bharat B Panda <bharat.b.panda@intel.com>
Change-Id: I2bf1227ccf35ec0e411f93d5ff30c665fa3d3005
---
 groups/device-specific/caas/framework_manifest.xml | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/groups/device-specific/caas/framework_manifest.xml b/groups/device-specific/caas/framework_manifest.xml
index 0d70e23..9c309b3 100644
--- a/groups/device-specific/caas/framework_manifest.xml
+++ b/groups/device-specific/caas/framework_manifest.xml
@@ -129,10 +129,9 @@
             <instance>default</instance>
         </interface>
     </hal>
-    <hal format="hidl">
+    <hal format="aidl" optional="true">
         <name>android.hardware.wifi.hostapd</name>
-        <transport>hwbinder</transport>
-        <version>1.2</version>
+        <version>1</version>
         <interface>
             <name>IHostapd</name>
             <instance>default</instance>
-- 
2.37.0

